port/unix/omrsysinfo.c

```diff
@@ -368,6 +368,31 @@ static int32_t retrieveAIXMemoryStats(struct OMRPortLibrary *portLibrary, struct
 static int32_t retrieveZOSMemoryStats(struct OMRPortLibrary *portLibrary, struct J9MemoryInfo *memInfo);
 #endif
 
+#if defined(J9ZOS390)
+#define CVTBASE  (*(struct cvt * __ptr32 * __ptr32)16)
+	struct cvt {
+		char cvtxx1 [604];
+		struct rmct *  __ptr32 cvtrmct;
+	};
+	struct rmct {
+		char rmctxx1[4];
+		struct iracct *  __ptr32 rmctcct;
+	};
+	struct iracct {
+		char cctxx1 [102];
+		short ccvutilp;    /* system CPU utilization */
+		char cctxx2 [16];
+		short ccvifact;    /* number of online IFAs (zAAPs) */
+		short ccvutila;    /* processor utilization */
+		char cctxx3 [4];
+		short ccvsupct;    /* number of online SUPs (zIIPs) */
+		char cctxx4 [14];
+		short ccvutili;    /* IFA (zAAP) utilization */
+		short ccvutils;    /* SUP (zIIP) utilization */
+	};      
```

Comment:
I think we should delete this block of code as it is already implemented in omrsimap.h [1]. Please reuse `J9CVT`, `J9RMCT`, and `J9CCT` structures to get to the CPU utilization data. There are examples in the codebase of how to use these structures if you search for the struct names. Once that is done let's remove this piece of code.

[1] https://github.com/eclipse/omr/blob/e7406af2360b1cecbf1018111685524224e28fde/port/zos390/omrsimap.h#L76-L86

Reply:
I am not sure code in sysinfo.c should rely on the omrsimap.h header. Also when adding new structures please use OMR and not J9.